.github/workflows/update-semconv-integration-branch.yml

```diff
@@ -0,0 +1,97 @@
+name: Update semconv integration branch
+
+on:
+  schedule:
+    # daily at 10:24 UTC
+    - cron: '24 10 * * *'
+  workflow_dispatch:
+
+jobs:
+  update-semconv-integration-branch:
+    runs-on: ubuntu-latest
+    if: github.repository == 'open-telemetry/opentelemetry.io'
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          # this is needed in order to do the rebase below
+          fetch-depth: 0
+          # this is needed in order to trigger workflows when pushing new commits to the PR
+          token: ${{ secrets.OPENTELEMETRYBOT_GITHUB_TOKEN }}
+
+      - name: Set environment variables
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          version=$(git branch -r \
+                        | grep '^ *origin/opentelemetrybot/semantic-conventions-v[0-9]+\.[0-9]+\.[0-9]+$' \
+                        | sed 's/ *origin\/opentelemetrybot\/semantic-conventions-//')
+
+          if [[ -z "$versions" ]]; then
+            latest_version=$(gh release view \
+                          --repo open-telemetry/semantic-conventions \
+                          --json tagName \
+                          --jq .tagName)
+            if [[ $latest_version =~ ^v([0-9]+)\.([0-9]+)\.[0-9]+ ]]; then
+              major="${BASH_REMATCH[1]}"
+              minor="${BASH_REMATCH[2]}"
+              version="v$major.$((minor + 1)).0"
+            else
+              echo "unexpected version: $latest_version"
+              exit 1
+            fi
+          fi
+
+          echo "VERSION=$version" >> $GITHUB_ENV
+          echo "BRANCH=opentelemetrybot/semantic-conventions-${version}" >> $GITHUB_ENV
+
+      - name: Checkout or create branch
+        run: |
+          if ! git ls-remote --exit-code --heads origin $BRANCH; then
+            git checkout -b $BRANCH origin/main
+            git push -u origin $BRANCH
+          else
+            git checkout $BRANCH
+          fi
+
+      - name: Use CLA approved github bot
+        run: |
+          git config user.name opentelemetrybot
+          git config user.email 107717825+opentelemetrybot@users.noreply.github.com
+
+      - name: Rebase
```

Comment:
Hmm, I'm not _sure_ that I'd want to automatically rebase the otel.io repo (that _is_ what this is doing, right?). My gut feeling is that we might want to do this by hand. Maybe introduce a script bool argument to control this? Or just omit it and we can do it fully manually.

Reply:
it's merging from main (no force pushing involved), I can remove it for now if you're prefer to test doing it by hand first

Reply:
I'd prefer rebasing, to keep the PR branch clean, and make it easier to cherry pick if/once we want to create a final PR.

But let's leave this as is for now. We can revisit later if necessary.